docs/reference/changelog.rst

```diff
@@ -78,6 +69,109 @@ Changelog
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
 
+8.4.0 (2024-Sep-10)
+-------------------
+
+.. note::
+
+   8.4.0 includes changes from the :ref:`7.5.6<7.5.6_changelog>` release.
+
+Core
+====
+
+* Fixes a bug where Snapcraft would fail to inject itself into the build
```

Comment:
> Fixes

These notes use a different tense than past releases. They should be consistent. Did we change the underlying guidelines for commit messages?

Reply:
We didn't.  I wanted to see what a changelog looks like in the indicative mood instead of imperative.  

I like inidicative better but I switched it back for consistency.

Reply:
I don't see a change?

Reply:
Whoops, forgot to push the commit.  Should be changed now